iputil/util.go

```diff
@@ -1,93 +0,0 @@
-package iputil
-
-import (
-	"encoding/binary"
-	"fmt"
-	"net"
-	"net/netip"
-)
-
-type VpnIp uint32
```

Comment:
leaving a note to consider bringing this back as `type VpnIp netip.addr` mostly as documentation, so we don't forget

Reply:
or at least `type VpnIp=netip.addr`